File: Java/AerisDemo/src/com/xweather/sdkdemo/java/demoaerisproject/MapOptionsLocalActivity.java
package com.xweather.sdkdemo.java.demoaerisproject;
import android.os.Bundle;
import android.widget.RadioGroup;

import com.aerisweather.aeris.maps.MapOptionsActivity;

public class MapOptionsLocalActivity extends MapOptionsActivity implements RadioGroup.OnCheckedChangeListener
{
    protected void onCreate(Bundle savedInstanceState) {
        this.setContentView(R.layout.activity_map_options);
        super.onCreate(savedInstanceState);
    }
}

File: Kotlin/AerisSdkDemo/app/src/main/java/com/xweather/sdkdemo/kotlin/view/map/MapOptionsLocalActivity.kt
package com.xweather.sdkdemo.kotlin.view.map

import android.os.Bundle
import android.widget.RadioGroup
import com.aerisweather.aeris.maps.MapOptionsActivity
import com.aerisweather.aeris.maps.R

/**
 * TODO Settings page - convert to Compose
 */
class MapOptionsLocalActivity : MapOptionsActivity(), RadioGroup.OnCheckedChangeListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        this.setContentView(R.layout.activity_map_options)
        super.onCreate(savedInstanceState)
    }
}
